.pre-commit-config.yaml

```diff
@@ -18,3 +18,8 @@ repos:
   rev: v0.2.2
   hooks:
     - id: ruff
+- repo: https://github.com/asottile/pyupgrade
```

Comment:
I don't think we want to run this in pre-commit. we'll just do these upgrades as they come up and make sense for us